### ISSUE TITLE
Feature request: Set branch to compare against

### ISSUE THREAD

**Issue by jasonbarry** [Daniel15/BuildSize#19]
Right now, it appears that the comparison is always with the `master` branch. It would be nice if you could compare with a given branch. 

For example, in my repo I've changed my default/base branch to be `dev`. I would like the artifact measurement to compare against the `dev` branch instead of `master`.

Thanks for your awesome work!

**Comment by Daniel15** [comment]
Yeah there's a todo for that in the code: https://github.com/Daniel15/BuildSize/blob/0e39cd8040350631b35a6e85129a2376a9ecdac2/app/Services/Build/AbstractBuildService.php#L73. I haven't seen many projects that change the default branch, but I agree that it would be good to have!
